stories/Example.test.tsx

```diff
@@ -1,19 +1,19 @@
 import { composeStories } from '@storybook/react';
-import { render, screen } from '@testing-library/react';
+import { screen } from '@testing-library/react';
 import * as stories from './Example.stories';
 
-describe('Example', () => {
-  const { True, False } = composeStories(stories);
+const { True, False } = composeStories(stories);
 
+describe('Example', () => {
   test('flag shoud be True', async () => {
-    render(<True {...True.args} />);
+    await True.run();
```

Comment:
This writing style seems to be the mainstream at present.
https://storybook.js.org/docs/writing-tests/import-stories-in-tests/stories-in-unit-tests

However, `--experimental-vm-modules` flag is required.😇